skymp5-server/cpp/unit/PartOne_UpdateLookTest.cpp

```diff
@@ -1,5 +1,9 @@
 #include "TestUtils.hpp"
 
+using namespace TestUtils;
```

Comment:
It's better to either rewrite usages to explicit usage of namespace or explicitly say `using` for used methods (or whatever is imported from there)